lang/cs/Org.Apache.REEF.Driver/Bridge/Events/ClosedContext.cs

```diff
@@ -40,52 +39,35 @@ public ClosedContext(IClosedContextClr2Java clr2java)
         }
 
         [DataMember]
-        public string InstanceId { get; set; }
+        public string InstanceId { get; private set; }
+
+        private IActiveContextClr2Java ParentContextClr2Java { get; set; }
```

Comment:
Again, I am at a loss myself :-) I did not want to touch those parts, as they are `private` anyhow.